src/lib/list/list.ts

```diff
@@ -46,6 +46,23 @@ export const _MatListItemMixinBase = mixinDisableRipple(MatListItemBase);
 })
 export class MatNavList extends _MatListMixinBase implements CanDisableRipple {}
 
+
+@Component({
+  moduleId: module.id,
+  selector: 'mat-button-list',
+  exportAs: 'matButtonList',
+  host: {
+    'role': 'button list',
```

Comment:
`role` corresponds to single value that defines the interaction pattern for an element: 
https://www.w3.org/WAI/PF/HTML/wiki/RoleAttribute

There isn't any role that corresponds to a list of actions, so this isn't necessary here.